demos/ensembles/ensemble-link-prediction-example.ipynb

```diff
@@ -1146,6 +1146,15 @@
     "![image.png](attachment:image.png)"
    ]
   },
```

Comment:
this still shows a broken link for me: https://github.com/stellargraph/stellargraph/blob/3e6a1356fdab02ea5e139ee457e8ce21ebacbd19/demos/ensembles/ensemble-link-prediction-example.ipynb

Reply:
Yeah, unfortunately I displaying "attachment" images doesn't work on Github. It does work on nbviewer and Read the Docs (and locally) though: https://nbviewer.jupyter.org/github/stellargraph/stellargraph/blob/3e6a1356fdab02ea5e139ee457e8ce21ebacbd19/demos/ensembles/ensemble-link-prediction-example.ipynb